lib/vtk/commands/socks/gov.va.socks.plist.erb

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This an erb template for the LaunchAgent which is configured by the setup command.